File: VB/MsaglHelpers/LayoutCalculators/ILayoutCalculator.vb
Imports Microsoft.Msagl.Core.Layout
Imports Microsoft.Msagl.Core.Routing

Namespace MsaglHelpers
    Public Interface ILayoutCalculator
        ReadOnly Property LayoutAlgorithmSettings() As LayoutAlgorithmSettings
        Sub CalculateLayout(ByVal geometryGraph As GeometryGraph)
    End Interface
End Namespace

File: CS/MsaglHelpers/LayoutCalculators/ILayoutCalculator.cs
using Microsoft.Msagl.Core.Layout;
using Microsoft.Msagl.Core.Routing;

namespace MsaglHelpers {
    public interface ILayoutCalculator {
        LayoutAlgorithmSettings LayoutAlgorithmSettings { get; }
        void CalculateLayout(GeometryGraph geometryGraph);
    }
}
